mrmustard/lab/gates.py

```diff
@@ -335,6 +335,9 @@ class BSgate(Parametrized, Transformation):
     It applies to a single pair of modes.
     One can optionally set bounds for each parameter, which the optimizer will respect.
 
+    N.B. the phase in BSgate(theta,phi,modes=[i,j]) is equivalent to
+    Rgate(-phi, modes=[j]) >> BSgate(theta) >> Rgate(phi, modes=[j])
+
```

Comment:
Use sphinx syntax to format code
```suggestion
    N.B. the phase in ``BSgate(theta,phi,modes=[i,j])`` is equivalent to
    ``Rgate(-phi, modes=[j]) >> BSgate(theta) >> Rgate(phi, modes=[j])``
```